### PR TITLE
Kitchen page improvements

### PR DESCRIPTION
**Pull request by heidar**
I've written up some guidelines around keeping tableware clean, and
using the dishwasher. I think this is important as the company grows and
I've noticed piles of dirty tableware and lack of glasses lately.

I've also added some information on using the bins. Sometimes they get
really full and no one is too good to take out the rubbish!

Please let me know if any of this needs amending because the new
cleaners take care of it. I'm not sure what the arrengement with them
is.

### CODE REVIEW

File: guides/office/kitchen.md [hunk 1]
```diff
@@ -9,9 +9,31 @@ We love our coffee at Made Tech. We have fresh beans regularly delivered to the
  - When grinding beans that no meetings are going on in the meeting room as it's very loud and can interrupt voice calls and conversations
  - When you're done making coffee that you tidy up any spillages so the kitchen is left in a clean state
 
-## Sink regime
+## Dirty glasses, cutlery, and crockery regime
 
-We all appreciate a clear and useable sink. Please ensure:
+We all appreciate a clean kitchen, with no dirty tableware, and a clear sink. This is why everyone should put their dirty tableware in the dishwasher.
```
Comment (chrisblackburn) [inline guides/office/kitchen.md]: There's a lot of tableware here.

How about "We all appreciate a clean and orderly kitchen and a clear sink..."

File: guides/office/kitchen.md [hunk 2]
```diff
@@ -9,9 +9,31 @@ We love our coffee at Made Tech. We have fresh beans regularly delivered to the
...
+ - When you're finished with any tableware that you rinse them and put them in the dishwasher
+ - When the dishwasher is full, then load it with a cleaning agent and start it
+ - When the dishwasher is running already, then wash your items in the sink with detergent instead
+ - When the dishwasher has finished and has clean items in it, empty it. Then put your dirty items inside!
+
+ Note: Please do not put your horse in the dishwasher.
+
+## Rubbish rules
+
+We are passionate about recycling at Made Tech, please familiarise yourself with which items should be recycled on [Southwark Council's](https://www.southwark.gov.uk/bins-and-recycling/recycling/what-goes-in-recycling) website.
```
Comment (chrisblackburn) [inline guides/office/kitchen.md]: As we're a commercial premise, Southwark Council aren't responsible for our waste collection. See "What is Mixed Recycling / Dry Mixed Recycling?" here: https://thefirstmile.co.uk/contact#faqs (unfortunately you can't direct link to the specific FAQ :-()

File: guides/office/kitchen.md (hunk 1 above)
Comment (heidar) [reply guides/office/kitchen.md]: Agreed, this one got out of hand, will fix!

File: guides/office/kitchen.md (hunk 2 above)
Comment (heidar) [reply guides/office/kitchen.md]: Ahhh! I'll have a look at this and update accordingly.

File: guides/office/kitchen.md [hunk 3]
```diff
@@ -9,9 +9,30 @@ We love our coffee at Made Tech. We have fresh beans regularly delivered to the
...
+Before using the dishwasher, please familiarise yourself with [accepted dishwasher loading conventions](https://www.wikihow.life/Load-a-Dishwasher).
+
+Please ensure:
+
+ - When you're finished with any tableware that you rinse them and put them in the dishwasher
+ - When the dishwasher is full, then load it with a cleaning agent and start it
+ - When the dishwasher is running already, then wash your items in the sink with detergent instead
+ - When the dishwasher has finished and has clean items in it, empty it. Then put your dirty items inside!
+
+ Note: Please do not put your horse in the dishwasher.
```
Comment (yaseminercan) [inline guides/office/kitchen.md]: Can we remove this note? probably more of an insider joke :D

Comment (cob16) [reply guides/office/kitchen.md]: sound advice IMO 😄

**Comment by craigjbass** [comment]
mergearoo?

**Comment by lukemorton** [comment]
Lets do it @heidar!

**Pull request merged**